Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM python:3
+
+RUN pip install cis_interface
+
+CMD [ "echo", "Usage: cisrun <yaml_file1> [yaml_file2] [yaml_file3] [...]" ]
```

Comment:
Normally the default command here would probably print the help prompt, but I couldn't find a `--help` or `-?` prompt for `cisrun`, and giving it zero arguments only yielded a Python stack trace.